mantle/kola/harness.go

```diff
@@ -886,6 +886,7 @@ type externalTestMeta struct {
 	Conflicts                 []string `json:"conflicts"                           yaml:"conflicts"`
 	AllowConfigWarnings       bool     `json:"allowConfigWarnings"                 yaml:"allowConfigWarnings"`
 	NoInstanceCreds           bool     `json:"noInstanceCreds"                     yaml:"noInstanceCreds"`
+	Description               string   `json:"description"                         yaml:"description"`
```

Comment:
I don't know Go, but while scrolling through the code I noticed that some strings (i.e distros) have `omitempty` implemented. Is that something worth considering here?

Reply:
I think, it is a way for go to say, if it doesn't have anything defined omits it in JSON file.
So instead of adding something like:
{ description = ""}  it will be {}
In other words don't add it in the JSON

Reply:
Thanks @ravanelli , can I understand that  `omitempty` is not necessary for this, as it is harmless?